hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -74,6 +74,20 @@ <h2>Status</h2>
     </tbody>
 </table>
 
+<h2>Meta-Data Volume Information</h2>
+<table class="table table-bordered table-striped" class="col-md-6">
+    <tbody>
+    <tr>
+        <td>Ratis Directory</td>
+        <td>{{$ctrl.overview.jmx.ScmRatisLogDirectory}}</td>
```

Comment:
Now that the method name is getRatisLogDirectory , I think you need to change it here as well , same for rocksdb dir

Reply:
Thanks it has been corrected